cmd/maestro/agent/cmd.go

```diff
@@ -38,17 +35,8 @@ func NewAgentCommand() *cobra.Command {
 	cmd.Short = "Start the Maestro Agent"
 	cmd.Long = "Start the Maestro Agent"
 
-	// check if the flag is already registered to avoid duplicate flag define error
-	if flag.CommandLine.Lookup("alsologtostderr") != nil {
-		flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
-	}
-
-	// add klog flags
-	klog.InitFlags(nil)
-
-	flags := cmd.Flags()
+	flags := cmd.PersistentFlags()
 	flags.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
-	flags.AddGoFlagSet(flag.CommandLine)
```

Comment:
where is to configure the logs if you remove it in agent?

Reply:
the flags are in the root command, we can still configure it when we run sub command(maestro migrate/server/agent).